bibliopixel/remote/control.py

```diff
@@ -165,9 +166,20 @@ def run_animation(self, name=None):
     def stop_animation(self, data):
         return self.run_animation()
 
+    def change_brightness(self, data):
+        if isinstance(data, str):
+            try:
+                data = int(data)
+            except:
+                return False, 'Invalid brightness value!'
```

Comment:
Exceptions are expensive. They are for when you can proceed no longer. The thing is that here I don't really care that I got a bad value, I'm just informing the idiot that passed in `twelve` for a value.
Yes, there are a few rare times when I'll throw an exception, handle it, and keep going... but exception != return error. Especially in an REST interface.